### PR TITLE
taxonomy: ingredients.txt

### PR DESCRIPTION
**Pull request by teolemon**
### What
taxonomy: ingredients.txt

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -27,6 +27,18 @@ synonyms:es:jarabe, sirope
...
+synonyms:de:halb-vollständig, halb-komplett
+
+synonyms:it:semi-completo, mezzo completo
+
+synonyms:pt:semi-completo, meio completo
+
+synonyms:nl:halfvol, half-compleet
+
+synonyms:ru:полу-полный, полу-комплексный
+
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:41]: Are you sure that it is a good idea?

In French, it is common to have both forms, they are synonyms.

For each other languages, none of both forms are found in the taxonomy...

**Review by benbenben2 — APPROVED** [review]
LGTM

See the comment

**Comment by aleene** [comment]
The dutch entry half-vol and half-compleet is not right as synonyms.

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -27,6 +27,18 @@ synonyms:es:jarabe, sirope
...
 
+synonyms:es:semi-completo, medio completo
+
+synonyms:de:halb-vollständig, halb-komplett
+
+synonyms:it:semi-completo, mezzo completo
+
+synonyms:pt:semi-completo, meio completo
+
+synonyms:nl:halfvol, half-compleet
```
Comment (teolemon) [inline taxonomies/ingredients.txt:38]:
```suggestion
```

File: taxonomies/ingredients.txt [hunk 3]
```diff
@@ -27,6 +27,18 @@ synonyms:es:jarabe, sirope
 
 synonyms:fr:semi-complet, demi-complet
 
+synonyms:es:semi-completo, medio completo
+
+synonyms:de:halb-vollständig, halb-komplett
+
+synonyms:it:semi-completo, mezzo completo
```
Comment (alexgarel) [inline taxonomies/ingredients.txt:34]: In italian it's "integrale", I don't think it's "completo".

And I dont think mezzo integrale exists…

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (teolemon) [inline taxonomies/ingredients.txt:41]:
```suggestion

```

**Comment by teolemon** [comment]
Fixed @benbenben2 @alexgarel

**Pull request merged**